connexion/json_schema.py

```diff
@@ -128,6 +128,29 @@ def nullable_validation_fn(validator, to_validate, instance, schema):
     return nullable_validation_fn
 
 
+def validate_required(validator, required, instance, schema):
+    if not validator.is_type(instance, "object"):
+        return
+
+    for prop in required:
+        if prop not in instance:
+            properties = schema.get('properties')
+            if properties is not None:
+                subschema = properties.get(prop)
+                if subschema is not None:
+                    if 'readOnly' in validator.VALIDATORS and subschema.get('readOnly'):
+                        continue
+                    if 'writeOnly' in validator.VALIDATORS and subschema.get('writeOnly'):
```

Comment:
Now I finally grasp that this line of code is validating a response, the field is marked as required, the field is also marked as writeOnly, and the function has detected that a response instance is MISSING the required writeOnly field. Adding the `continue` causes this condition to be ignored. Is this another subtle JSON schema validation thing?